internal/ctrlutils/ingress-status.go

```diff
@@ -272,11 +279,15 @@ func UpdateIngressLegacy(
 				break
 			}
 			if errors.IsNotFound(err) {
+				log.Debugf("failed to update Ingress V1 status because the object (%s/%s) is gone, status update stopped.", namespace, name)
 				return nil
 			}
-
-			log.Errorf("failed to update Ingress V1beta1 status. %v. retrying...", err)
-			time.Sleep(time.Second)
+			if errors.IsConflict(err) {
+				log.Debugf("failed to update Ingress V1Beta1 status because the object (%s/%s) changed. retrying...", namespace, name)
```

Comment:
```suggestion
				log.Debugf("failed to update Ingress V1Beta1 status (%s/%s) status, conflict %v err", namespace, name, err)
```

Reply:
:+1: changed in 2812019f36e6aed24e3021ec977f0475ab68e90d